src/test/enhetstester/kotlin/no/nav/familie/ba/sak/kjerne/steg/VurderTilbakekrevingStegTest.kt

```diff
@@ -128,6 +137,34 @@ class VurderTilbakekrevingStegTest {
         )
     }
 
+    @Test
+    fun `skal ikke utføre steg for migreringsbehandling med endre migreringsdato når det finnes etterbetaling mindre enn maks beløp`() {
```

Comment:
```suggestion
    fun `skal utføre steg for migreringsbehandling med endre migreringsdato når det finnes etterbetaling mindre enn maks beløp`() {
```
Hvis jeg leser testen og favro riktig så skal det ikke stå "ikke" her